packages/paste-libraries/use-clipboard/src/index.tsx

```diff
@@ -0,0 +1,2 @@
+export {useClipboard} from 'use-clipboard-copy';
+export type {UseClipboardOptions as UseClipboardProps} from 'use-clipboard-copy';
```

Comment:
❓ Are these the only types we need to export?

Reply:
good point, i'll also export the return value of `useClipboard` 👍